src/yunohost/backup.py

```diff
@@ -1924,6 +1920,12 @@ def mount(self, restore_manager):
                          self._archive_file, exc_info=1)
             raise YunohostError('backup_archive_open_failed')
 
+        try:
+            files_in_archive = tar.getnames()
+            print(files_in_archive)
+        except IOError as e:
+            raise YunohostError("backup_archive_corrupted", archive=self._archive_file, error=str(e))
```

Comment:
>         Exceptions:
>        backup_archive_open_failed -- Raised if the archive can't be open
>        """

You can add this exception to the comments, and `backup_archive_cant_retrieve_info_json` as it may be raised bellow